yubikit/openpgp.py

```diff
@@ -955,6 +988,8 @@ def _pad_message(attributes, message, hash_algorithm):
 
 
 class OpenPgpSession:
+    """A session with the OpenPGP appliation."""
```

Comment:
Typo in application

Reply:
Pushed a commit that resolves this and a couple of other typos I found.